Eatery/Views/Eateries/MealItemTableViewCell.swift

```diff
@@ -21,12 +24,66 @@ class MealItemTableViewCell: UITableViewCell {
         addSubview(nameLabel)
         nameLabel.translatesAutoresizingMaskIntoConstraints = false
         nameLabel.snp.makeConstraints { make in
-            make.edges.equalTo(self).inset(UIEdgeInsets(top: 8, left: 16, bottom: 8, right: 16))
+            make.edges.equalToSuperview().inset(UIEdgeInsets(top: 4, left: 16, bottom: 4, right: 16))
+        }
+        
+        favoriteButton.addTarget(self, action: #selector(tappedFavorite), for: .touchUpInside)
+        favoriteButton.isHidden = true
+        addSubview(favoriteButton)
+        favoriteButton.translatesAutoresizingMaskIntoConstraints = false
+        favoriteButton.snp.makeConstraints { make in
+            make.top.bottom.equalToSuperview()
+            make.trailing.equalToSuperview().inset(16)
+            make.width.equalTo(favoriteButton.snp_width)
```

Comment:
do you need to do this?

Reply:
Yes

Reply:
Why?

Reply:
I think I might be confused as to what's being referred to as potentially removable? If you remove this constraint code the favorite button won't even show up? Should I be constraining favoriteButton in a different way?